WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt

```diff
@@ -28,6 +28,7 @@ has_romance_culture_trigger = {
 		culture = helleno_thracian
 		culture = massalian
 		culture = romano_dacian
+		culture = spartan
```

Comment:
No, not at all. The Roman Empire can't be usurped and it has no connection to Ancient Greeks.